Bluepill-cli/BPInstanceTests/BPRegexTests.m

```diff
@@ -19,8 +19,6 @@ @implementation RegexTests
 
 - (void)setUp {
     [super setUp];
```

Comment:
I think these methods are auto-generated by Xcode when you add a new test. I guess we could remove them but they do serve as a placeholder for setup code.